File: udemy/pascal/projetos/Repeticao_10pares.pas
program Repeticao_10pares;
var 
    n,cont_par, i: integer;
    
begin
cont_par := 1;
n := 0;
  while(cont_par <= 10) do 
    begin
    n := n+1;
    if (n mod 2 = 0) then
        begin
            cont_par := cont_par + 1;
            writeln('O numero ',n, ' eh par');
        end;
    end;
end.

File: udemy/java/Repeticao_10pares/src/repeticao_10pares/Repeticao_10pares.java
package repeticao_10pares;
import javax.swing.JOptionPane;
public class Repeticao_10pares {
	public static void main(String[] args) {
		Integer cont_par, n;
		cont_par = 1;
		n = 0;
		String mensagem="";
		while(cont_par <= 10) {
			n++;
			if(n%2 ==0) {
				cont_par++;
				mensagem =  mensagem + n+" é um numero par\n";
			}
		}
		JOptionPane.showMessageDialog(null, mensagem);
	}

}
